org_fedora_oscap/content_discovery.py

```diff
@@ -175,23 +175,26 @@ def finish_content_fetch(self, fetching_thread_name, fingerprint, report_callbac
 
     def _verify_fingerprint(self, dest_filename, fingerprint=""):
         if not fingerprint:
-            log.info("No fingerprint provided, skipping integrity check")
+            log.info("OSCAP Addon: No fingerprint provided, skipping integrity check")
             return
 
         hash_obj = utils.get_hashing_algorithm(fingerprint)
         digest = utils.get_file_fingerprint(dest_filename,
                                             hash_obj)
         if digest != fingerprint:
             log.error(
+                "OSCAP Addon: "
                 f"File {dest_filename} failed integrity check - assumed a "
                 f"{hash_obj.name} hash and '{fingerprint}', got '{digest}'"
             )
-            msg = _(f"Integrity check of the content failed - {hash_obj.name} hash didn't match")
+            msg = _(f"OSCAP Addon: Integrity check of the content failed - {hash_obj.name} hash didn't match")
             raise content_handling.ContentCheckError(msg)
         log.info(f"Integrity check passed using {hash_obj.name} hash")
 
     def _finish_actual_fetch(self, wait_for, fingerprint, report_callback, dest_filename):
+        log.info(f"OSCAP Addon: Waiting for thread {wait_for}")
```

Comment:
I can see messages in the log: `Waiting for thread None` and `Finished waiting for thread None`.